docs/csharp/whats-new/index.md

```diff
@@ -14,6 +14,12 @@ ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
 
 # What's new in C# #
 
+This page provides a roadmap of new features in each major release of
+the C# language. The links below provide detailed information on the
+major features added in each release.
+
+> [!IMPORTANT]
+> The C# language relies on types and methods in a *standard library* for some of the features. To use the latest language features in older environments, you may need to install specific libraries. These are documented in the page for each specific version. You can learn more about the [relationships between language and library](relationships-between-language-and-library.md) for background on this dependency.
```

Comment:
I think this needs to be phrased a bit more strongly to make it clear that this isn't just about the compiler resolving type and method references, but about those types being used internally by the compiler for multiple language features. An example would help, and the best is probably Exception -- the exception handling system relies on the presence of System.Exception.